### PR TITLE
[CI] Windows Release and Signing

### PR DESCRIPTION
**Pull request by flavioislima**
This sets a Windows release workflow that will be triggered with the others and automatically sends the binary to be signed on signpath and then gets back the artifacts and updates the latest.yml with the new information and creates or updates the release.

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: yarn.lock
Comment (BrettCleary) [inline yarn.lock:1]: This is a lot of updates. Is this from `yarn upgrade`? 

Looks like these dependency updates are causing the builds to fail too. 

Maybe we should revert to the previous lockfile and handle these updates on a separate branch

**Review by BrettCleary — CHANGES_REQUESTED** [review]
GHA looks good. All the builds are failing now though. I think it's due to the yarn.lock updates so maybe we should revert those

File: yarn.lock
Comment (flavioislima) [reply yarn.lock:1]: Fixed

**Review by BrettCleary — APPROVED** [review]
LGTM

**Review by eliobricenov — APPROVED** [review]

**Pull request merged**